docs/source/markdown/podman-network.1.md

```diff
@@ -41,10 +41,12 @@ Podman requires specific default IPs and, thus, network subnets.  The default va
 ### Podman network
 The default bridge network (called `podman`) uses 10.88.0.0/16 as a subnet. When Podman runs as root, the `podman` network is used as default.  It is the same as adding the option `--network bridge` or `--network podman`. This subnet can be changed in **[containers.conf(5)](https://github.com/containers/common/blob/main/docs/containers.conf.5.md)** under the [network] section. Set the `default_subnet` to any subnet that is free in the environment. The name of the default network can also be changed from `podman` to another name using the default network key. Note that this is only done when no containers are running.
 
-### Slirp4netns
-When Podman is run as rootless, the internet connectivity is provided with slirp4netns by default. Slirp4nents uses 10.0.2.0/24 for its default network. This can also be changed in **[containers.conf(5)](https://github.com/containers/common/blob/main/docs/containers.conf.5.md)** but under the `[engine]` section. Use the `network_cmd_options` key and add `["cidr=X.X.X.X/24"]` as a value. Note that slirp4netns needs a network prefix size between 1 and 25. This option accepts an array, so more options can be added in a comma-separated string as described on the **[podman-network-create(1)](podman-network-create.1.md)** man page. To change the CIDR for just one container, specify it on the cli using the `--network` option like this: `--network slirp4netns:cidr=192.168.1.0/24`.
+### Pasta
+The default rootless networking tool has been switched from **[slirp4netns](https://github.com/rootless-containers/slirp4netns)** to **[pasta](https://passt.top/passt/about/#pasta-pack-a-subtle-tap-abstraction)**.
```

Comment:
This sentence makes no sense like this, either add when it was changed (i.e. podman 5.0) or just remove it altogether